### ISSUE TITLE
Unnecessary getImageData call

### ISSUE THREAD

**Issue by khokm** [mapbox/tiny-sdf#45]
https://github.com/mapbox/tiny-sdf/blob/bd847e192f5b147d644e4fad3e0cf45e31166ea3/index.html#L253
https://github.com/mapbox/tiny-sdf/blob/bd847e192f5b147d644e4fad3e0cf45e31166ea3/index.html#L256

`gl.texImage2D` supports `HTMLCanvasElement` as source. So there is no need to call `getImageData` if you want to load entire canvas data.

**Comment by mourner** [comment]
Good catch! Although it's a demo page so it doesn't matter much, and it was done this way because canvas can only be passed in WebGL 2 contexts, and support for it wasn't wide at the time of making this page.
